disperser/dataapi/docs/swagger.yaml

```diff
@@ -343,6 +343,31 @@ paths:
       summary: Fetch throughput time series
       tags:
       - Metrics
+  /operatorsInfo/deregisteredOperators:
```

Comment:
To be consistent with the rest endpoints, can you name this with underscore? `/operator/deregistered_operators` for example.

Reply:
This is not resolved

Reply:
Why does operatorsInfo need a special format?

Reply:
The path naming of `/operatorsInfo/deregistered_operators`. Is it a good practice to use camel case? Is it at least consistent with existing naming sytle?